packages/services/src/services/Analytics/Analytics.js

```diff
@@ -0,0 +1,56 @@
+import root from 'window-or-global';
+/**
+ * Analytics API class with methods for firing analytics events on
+ * ibm.com
+ */
+class AnalyticsAPI {
+  /**
+   * service to fire a stats/metrics event for an action
+   *
+   * @param {object} eventData Object with standard IBM metric event properties and values to send to Coremetrics
+   * @returns {object} JSX object
+   * 
+   * example of eventData object expected:
```

Comment:
@annawen1 can this be placed in an `@example`? How does this generate in JSDocs?

Reply:
done